src/amo/components/ReportAbuseButton/index.js

```diff
@@ -149,13 +149,13 @@ export class ReportAbuseButtonBase extends React.Component<Props> {
             className="ReportAbuseButton-show-more Button--report Button--fullwidth"
             onClick={this.showReportUI}
           >
-            {i18n.gettext('Report this add-on for abuse')}
+            {i18n.gettext('Report an issue with this add-on')}
```

Comment:
As @jvillalobos beat me to mentioning (in https://github.com/mozilla/addons-frontend/issues/4025#issuecomment-349103373), this isn't the right wording–it implies this is used as a mechanism to report bugs and will be surfaced to the developer, but neither are true. Sadly the old wording has to come back (even though I like it less).